inject/src/main/java/io/micronaut/context/Qualifier.java

```diff
@@ -65,4 +66,41 @@ default boolean contains(Qualifier<T> qualifier) {
     default <BT extends BeanType<T>> Optional<BT> qualify(Class<T> beanType, Stream<BT> candidates) {
         return reduce(beanType, candidates).findFirst();
     }
+
+    /**
+     * Check if the candidate qualifies.
+     *
+     * @param beanType  The bean type
+     * @param candidate The candidate
+     * @return true if matches
+     * @since 4.2.0
+     */
+    default boolean isQualifies(Class<T> beanType, BeanType<T> candidate) {
```

Comment:
maybe doesQualify? @graemerocher naming suggestions please :D

Reply:
either `doesQualifiy` is `isQualified`

Reply:
renamed